libs/application/templates/children-residence-change/src/fields/Overview/SignatureModal/index.tsx

```diff
@@ -102,35 +67,29 @@ const SignatureModal = ({
         )}
         <Box
           className={cn(styles.controlCodeContainer.general, {
-            [styles.controlCodeContainer.upload]: isUpload,
-            [styles.controlCodeContainer.error]: hasError,
+            [styles.controlCodeContainer.hide]: hasError,
           })}
-          marginTop={4}
-          background={
-            statusVariantStyles[fileSignatureState.content.type].background
-          }
         >
-          {!isRequest &&
-            (isUpload ? (
-              <>
-                <Text lineHeight="xl" variant="small" marginTop={2}>
-                  {formatMessage(signatureModal.security.numberLabel)}
-                </Text>
-                <Text lineHeight="sm" variant="h1">
-                  {controlCode}
-                </Text>
-              </>
-            ) : (
-              <Icon
-                className={styles.iconContainer}
-                color={
-                  statusVariantStyles[fileSignatureState.content.type].iconColor
-                }
-                icon={statusVariantStyles[fileSignatureState.content.type].icon}
-                size="large"
-                type="filled"
-              />
-            ))}
+          {isUpload ? (
+            <>
+              <Text lineHeight="xl" variant="small">
+                {formatMessage(signatureModal.security.numberLabel)}
+              </Text>
+              <Text lineHeight="sm" variant="h1">
+                {controlCode}
+              </Text>
+            </>
+          ) : (
+            <Icon
+              className={cn(styles.iconContainer.general, {
+                [styles.iconContainer.hideIcon]: isRequest,
+              })}
+              color="blue400"
+              icon="checkmarkCircle"
+              size="large"
+              type="filled"
+            />
+          )}
```

Comment:
I could be wrong, but we only show the icon when the request is successful, right? If so should we change this to the following: 
```          {isUpload && (
             <>
               <Text lineHeight="xl" variant="small">
                 {formatMessage(signatureModal.security.numberLabel)}
               </Text>
               <Text lineHeight="sm" variant="h1">
                 {controlCode}
               </Text>
             </>
           )}
             <Icon
               className={cn(styles.iconContainer.general, {
                 [styles.iconContainer.hideIcon]: fileSignatureState.status === FileSignatureStatus.SUCCESS,
               })}
               color="blue400"
               icon="checkmarkCircle"
               size="large"
               type="filled"
             />
           }```

Reply:
I think there is something missing from this suggestion? 🤔